packages/reputation-miner/viz/README.md

```diff
@@ -0,0 +1,15 @@
+# Welcome to the Colony Mining visualizers!
```

Comment:
I'd argue we remove this readme and incorporate information on running the visualiser in the network docs for the miner. Otherwise this feels too separate from the client and I'd prefer to think of them as the UI and logic to the same thing.

Reply:
Ok, I can move the readme.